node/src/Commands.ts

```diff
@@ -1847,11 +1978,20 @@ export function createGeoAdd(
     let args: string[] = [key];
 
     if (options) {
-        args = args.concat(options.toArgs());
+        if (options.updateMode) {
+            args.push(options.updateMode);
+        }
+
+        if (options.changed) {
+            args.push("CH");
```

Comment:
do we have a place to put string constants?

Reply:
Not currently, I think if we want to do this, it should be a part of a different PR as we may want to convert other types with string constants to enums. Would potentially be big enough on it's own.